config/locales/custom.en.yml

```diff
@@ -131,11 +131,14 @@ en:
     share: "Share your list of your recommended investment projects with this link:"
   welcome:
     budgets:
-      title: "100 million to make Madrid"
+      title_html: "100 million<br> to make Madrid"
       text_html: "In the final vote of the participatory budgets 2017, you decide the citizen projects that will be included in the City Council's budget <strong>in the city and the districts</strong>."
       date: "From May 15 to June 30"
       button: "Vote projects"
       more_info: "More information about participatory budgets"
+      chincheto:
+        title: "Watch SrChincheto77 video on YouTube (opens in new tab)"
+        alt: "SrChincheto77 come with us for the Madrid that decides the citizenship!"
```

Comment:
Maybe `accompanies us` could be a better translation, instead of [`come with us`](https://www.youtube.com/watch?v=F30zbHuTrh0)  (anyway I think it should be `comes` instead of `come`) right?